ci/recipe/meta.yaml

```
@@ -32,6 +32,8 @@ requirements:
     - q2templates {{ release }}.*
     - q2-sample-classifier {{ release }}.*
     - q2-feature-table {{ release }}.*
+    - matplotlib < 3.3
+    - matplotlib-base < 3.3
```

https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications

> Remember that the version specification cannot contain spaces, as spaces are used to delimit the package, version, and build string in the whole match specification. python >= 2.7 is an invalid match specification. Furthermore, python>=2.7 is matched as any version of a package named python>=2.7.

```suggestion
    - matplotlib <3.3
    - matplotlib-base <3.3
```